cmd/statusd/utils.go

```diff
@@ -1414,6 +1418,29 @@ func testJailFunctionCall(t *testing.T) bool {
 	return true
 }
 
+func testNodeOffline(t *testing.T) bool {
+
+	StopNode()
+	//time to sync
+	time.Sleep(5 * time.Second)
```

Comment:
Do we need this? Is there a more reliable way? It has great potential to result in a flaky test.